test/classes/bradc/bitOpOnClass.chpl

```diff
@@ -1,29 +1,32 @@
 class MyClass {
-var len: int;
-var dom: domain(1);
-var arr: [dom] int;
-proc MyClass(length: int) {
-  this.len = length;
+  var len: int;
+  var dom: domain(1);
+  var arr: [dom] int;
+
+  proc init(length: int) {
+    this.len = length;
     this.dom = {1..length};
-      forall (a,i) in zip(this.arr, 13..) do
-        a = i;
-        }
+    super.init();
```

Comment:
I think I called that out in the commit message, but did move the commit messages into the PR message - I'll do that now, thanks!